src/Entity/Image.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\ImageRepository;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Blameable\Traits\BlameableEntity;
+use Gedmo\SoftDeleteable\Traits\SoftDeleteableEntity;
+use Gedmo\Timestampable\Traits\TimestampableEntity;
+
+#[ORM\Entity(repositoryClass: ImageRepository::class)]
+class Image
+{
+    use TimestampableEntity;
+    use SoftDeleteableEntity;
+    use BlameableEntity;
+
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private ?int $id = null;
+
+    #[ORM\Column(type: Types::TEXT)]
+    private ?string $source = null;
+
+    #[ORM\Column(length: 255, nullable: true)]
+    private ?string $local = null;
+
+    #[ORM\OneToOne(mappedBy: 'image', cascade: ['persist', 'remove'])]
+    private ?Event $event = null;
+
+    #[ORM\Column]
+    private ?bool $updated = false;
```

Comment:
What is this property used for? A little comment may come in handy.

Reply:
Removed